webpack/scenes/ContentViews/Details/Filters/CVRpmFilterContent.js

```diff
@@ -97,7 +100,26 @@ const CVRpmFilterContent = ({ filterId, inclusion }) => {
             variant={TableVariant.compact}
             autocompleteEndpoint={`/content_view_filters/${filterId}/rules/auto_complete_search`}
             fetchItems={useCallback(params => getCVFilterRules(filterId, params), [filterId])}
-          />
+            additionalListeners={[(ruleSaved)]}
+          >
+            <Split hasGutter>
+              <SplitItem>
+                <Button onClick={openAddRpmRuleModal} variant="secondary" aria-label="create_rpm_rule">
+                  {__('Add RPM rule')}
+                </Button>
+              </SplitItem>
+            </Split>
+            {addRpmRuleModalOpen &&
```

Comment:
Again, I'm a fan of the: 
if (!show) return null; 
at the top of the functional scope.
instead of double conditions

Reply:
That sounds amazing for readability, +1

Reply:
Rebased and changed this to remove the show param from the Modal as the settled pattern for modals..